node/src/database/db_initializer.rs

```diff
@@ -727,7 +727,8 @@ mod tests {
         let mut stmt = conn
             .prepare("select name, value, encrypted from config")
             .unwrap();
-        let _ = stmt.query_map([], |_| Ok(42)).unwrap();
+        // TODO: The next line's execution returns Err(ExecuteReturnedResults). Don't know why.
```

Comment:
I think execute in the rusqlite library is meant to return only descriptive numbers not any data itself. 
If you're employing an SQL statement which is supposed to return a value (the one stored in the database column), you need to use `query`-like methods. For example `query_map`.